### PR TITLE
Noted that you can redeem coupons

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://trello.com/c/R6baPpd7/929-3-consume-coupons

@lancejjohnson PTAL

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@lancejjohnson PTAL

**Comment by ahardin-rh** [comment]
@lancejjohnson @abhgupta I made some changes based on the information in https://github.com/openshift/online-registration/pull/958. PTAL

**Comment by abhgupta** [comment]
@ahardin-rh There is a meeting scheduled for Monday (2/19) to discuss the user experience around specifying these coupons. The documentation for this feature may change slightly based on the outcome of that meeting. Lance and I will keep you posted. If you'd like to attend that meeting, let us know.

**Comment by ahardin-rh** [comment]
@abhgupta Sounds great. Yeah, you can add me and I will try to make it. If not, we can iron out details afterward. Thanks!

File: getting_started/choose_a_plan.adoc [hunk 1]
```diff
@@ -57,6 +57,22 @@ then your account will be terminated and you will be banned from creating
...
+You can also redeem coupons after the registration process:
+
+. From the *Active Subscriptions* page, click *Manage Subscriptions*.
+
+. On the next *Subscriptions* page, click *Manage* next to the *Coupons* heading.
+
+. On the next *Subscription Coupons* page, you will see (if applicable) coupons
+that are available for you to use and those that are already in use. To apply a
+new coupon, click *Apply New Coupon* and complete the workflow.
+
```
Comment (yasun1) [inline getting_started/choose_a_plan.adoc]: Currently, the coupon index page is out of date, and has been changed. There is no applicable coupons shown for the user to apply, the user only can see the applied coupons.

Comment (lancejjohnson) [reply getting_started/choose_a_plan.adoc]: @ahardin-rh We just need to remove "that are available for you to use and those"

**Review by lancejjohnson — CHANGES_REQUESTED** [review]

File: getting_started/choose_a_plan.adoc (hunk 1 above)
Comment (ahardin-rh) [reply getting_started/choose_a_plan.adoc]: @yasun1 @lancejjohnson Updated. Thanks!

**Comment by yasun1** [comment]
LGTM

**Comment by ahardin-rh** [comment]
@adellape Thanks for cherry-picking while I was out! :beers: You're a pal.

**Pull request merged**